docs/install/releases.md

```diff
@@ -7,10 +7,9 @@ We recommend enterprise customers test the compatibility of new releases with
 their infrastructure on a staging environment before upgrading a production
 deployment.
 
-We support two release channels:
-[mainline](https://github.com/coder/coder/2.10.0) for the edge version of Coder
-and [stable](https://github.com/coder/coder/releases/latest) for those with
-lower tolerance for fault. We field our mainline releases publicly for one month
+We support two release channels: [mainline](https://github.com/coder/coder/releases/v2.10.1)
+for the edge version of Coder and [stable](https://github.com/coder/coder/releases/latest)
```

Comment:
Should we call the `mainline` edge? As we also have `ghcr.io/coder/coder-preview:main`, which points to the tip of the `main` branch